AndelaOSP/Andela-Socials
# #159402806: Allow users to upload an image when creating an event 🖼

#### What Does This PR Do? 

Implement Functionality for allowing a user to attach a featured image when creating an event  

#### Description Of Task To Be Completed

When a user is creating an event, they should have the capacity to attach an image and the image should be uploaded to Google Cloud Storage.

- [x] Implement mutation to send request to GCP
- [x] Send mutation to create image
- [x] Create actions and reducers to handle image sending

#### How can this be manually tested?

Assuming you have the app setup following the [README.md](https://github.com/AndelaOSP/Andela-Socials/blob/develop/README.md).

- Start the application
- Navigate to `localhost:9000`
- You should be able to see a button to create an event then try creating one(See gif below 👇 )  

#### What are the relevant pivotal tracker stories?

[#159402806](https://www.pivotaltracker.com/n/projects/2174978/stories/159402806)

#### Screenshot(s)

![demo_image](https://user-images.githubusercontent.com/19430799/45952316-4cc7b900-c00f-11e8-809d-c6d3e3c76b23.gif)
